tests/1-login.spec.ts

```diff
@@ -0,0 +1,19 @@
+import { test, expect } from '@playwright/test';
+
+const user = {
+  name: 'Jane Doe',
+  email: 'customer@practicesoftwaretesting.com',
+  password: 'welcome01'
+}
+
+test('login', async ({ page }) => {
+  await page.goto('https://practicesoftwaretesting.com/auth/login');
```

Comment:
Let's add app URL to the playwright config - https://playwright.dev/docs/test-webserver#adding-a-baseurl